apps/ledger-live-mobile/src/newArch/features/Market/hooks/usePullToRefresh.ts

```diff
@@ -1,15 +1,24 @@
 import { useState, useCallback, useEffect } from "react";
+import { track } from "~/analytics";
 
 interface UsePullToRefreshProps {
   loading: boolean;
   refresh: () => void;
 }
+
 function usePullToRefresh({ loading, refresh }: UsePullToRefreshProps) {
+  const trackPullToRefresh = useCallback(() => {
+    track("button_clicked", {
+      button: "pull to refresh",
+    });
+  }, []);
+
   const [refreshControlVisible, setRefreshControlVisible] = useState(false);
   const handlePullToRefresh = useCallback(() => {
     refresh();
     setRefreshControlVisible(true);
-  }, [refresh]);
+    trackPullToRefresh();
+  }, [refresh, trackPullToRefresh]);
```

Comment:
does this list need to be exhausted? given `trackPullToRefresh` is static?

Reply:
I added it to avoid lint error but yes it might not be useful

Reply:
You can add this comment above the line to ignore the lint requirement. 
```suggestion
  }, 
    // eslint-disable-next-line react-hooks/exhaustive-deps
    [refresh]
  );
```